src/main/java/Homework1/Main.java

```diff
@@ -0,0 +1,26 @@
+package Homework1;
+
+import java.util.Scanner;
+
+public class Main {
+
+
+    public static void main(String[] args) {
+
+
+        Scanner scanner = new Scanner(System.in);
```

Comment:
Please use args from the start of the application. No need to complicate it for now

Reply:
It sais it can't automatically merge and that I have to create a pull request which I seem to have done.

Reply:
And I also can't delete the old Readme file form the src directory. I think I need your approval.